website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -11,7 +11,7 @@ dbt "adapters" are responsible for _adapting_ dbt's functionality to a given dat
 2. In the middle: A set of *macros* responsible for generating SQL that is compliant with the target database.
 3. (Optional) At the highest level: A set of *<Term id="materialization">materializations</Term>* that tell dbt how to turn model files into persisted objects in the database.
 
-This guide will walk you through the first two steps, and provide some resources to help you validate that your new adapter is working correctly.
+This guide will walk you through the first two steps, and provide some resources to help you validate that your new adapter is working correctly. Once you feel it is mostly working, please let the community know that is available to use by adding the adapter to the [Available Adapters](/website/docs/docs/available-adapters.md) page by following the steps given in [Documenting your adapter](/website/docs/docs/contributing/documenting-a-new-adapter.md).
```

Comment:
> Once you feel it is mostly working
Could we be more...specific here? This sounds very confusing.

Reply:
good call. improved!